js/form-validation.js

```diff
@@ -1,10 +1,36 @@
 /* eslint-disable linebreak-style */
+const nameField = document.getElementById('name');
+const emailField = document.getElementById('email');
+const commentField = document.getElementById('comment');
+// This is the local storage object that contains all the data
+const inputData = {
+  name: '',
+  email: '',
+  comment: '',
+};
+
+// If the local storage has values then set the value of input
+window.addEventListener('load', () => {
+  // Retrive the JSON string
+  const jsonString = localStorage.getItem('inputData');
+  // Parse the JSON string back to JS object using JSON
+  const retriveObject = JSON.parse(jsonString);
+  // Add input data to input fields
+  nameField.value = retriveObject.name;
+  emailField.value = retriveObject.email;
+  commentField.value = retriveObject.comment;
+});
+
 document.getElementById('form').addEventListener('submit', (event) => {
-  const emailField = document.getElementById('email').value;
-  if (emailField.toLowerCase() === emailField) {
+  if (emailField.value.toLowerCase() === emailField.value) {
+    inputData.name = nameField.value;
+    inputData.email = emailField.value;
+    inputData.comment = commentField.value;
+    localStorage.setItem('inputData', JSON.stringify(inputData));
     return true;
```

Comment:
- [ ] The data is saved to the local storage when I submit the form, but the data should be saved on every input change as requested in the project requirements. I would suggest using the `change` event instead of the `submit` event.
![image](https://user-images.githubusercontent.com/83803414/189095520-20810363-35d3-4d1e-9e58-0c98d45a6066.png)